src/components/ClaimMasterPanelExt.js

```diff
@@ -40,13 +43,38 @@ class ClaimMasterPanelExt extends Component {
     this.props.clearLastClaimAt();
   }
 
+  getPolicyStatusLabel(timeDelta){
+    let policyStatusLabel = null;
+    if (timeDelta >= 0){
+      policyStatusLabel = "ClaimMasterPanelExt.InsureePolicyEligibilitySummaryActive.header";
+    }
+    else{
+      policyStatusLabel = "ClaimMasterPanelExt.InsureePolicyEligibilitySummaryInactive.header";
+    }
+    return policyStatusLabel;
+  }
+
+  getPolicyStatusLabelStyle(timeDelta, classes){
+    let policyStatusLabelStyle = null;
+    if (timeDelta >= 0){
+      policyStatusLabelStyle = classes.activeLabel;
+    }
+    else{
+      policyStatusLabelStyle = classes.inactiveLabel;
+    }
+    return policyStatusLabelStyle;
+  }
+
   render() {
     const { classes, claim, fetchingLastClaimAt, errorLastClaimAt, fetchedLastClaimAt, lastClaimAt } = this.props;
+    let timeDelta = getTimeDifferenceInDaysFromToday(this.props?.currentPolicy ? this.props?.currentPolicy[0]?.expiryDate : null);
+    let policyStatusLabel = this.props.currentPolicy ? this.getPolicyStatusLabel(timeDelta) : "ClaimMasterPanelExt.InsureePolicyEligibilitySummary.header";
```

Comment:
Use const instead of let. Moreover, it would be the best to create constant for this long string.